app/screens/home/channel_list/servers/servers_list/server_item/options/index.tsx

```diff
@@ -46,6 +52,33 @@ const ServerOptions = ({progress, server}: Props) => {
         console.log('ON Remove');
     }, [server]);
 
+    let sessionComponent;
+    if (isLoggedIn) {
```

Comment:
Does it make sense to use the same component and just change the props passed based on `isLoggedIn`?

Reply:
we can change it, but I don't see any major benefits.